python/pyngraph/ops/fused/clamp.cpp

```diff
@@ -17,14 +17,15 @@
 #include <pybind11/pybind11.h>
 #include <pybind11/stl.h>
 
-#include "ngraph/op/fused/elu.hpp"
-#include "pyngraph/ops/elu.hpp"
+#include "ngraph/op/fused/clamp.hpp"
+#include "pyngraph/ops/fused/clamp.hpp"
 
 namespace py = pybind11;
 
-void regclass_pyngraph_op_Elu(py::module m)
+void regclass_pyngraph_op_Clamp(py::module m)
```

Comment:
I'm not sure but it looks like the `Elu` bindings will be removed now. This file is required for `Clamp` but you should create a similar .cpp file for elu.

Reply:
Accidentally, I converted the original file `elu.cpp` to `clamp.cpp` instead of its copy file. There is `elu.cpp` in the `fused` package.